terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -1,17 +1,9 @@
 environment = "live"
 aws_profile = "live-eu-west-2"
 
-api_url = "https://private-api.companieshouse.gov.uk"
-cdn_host = "https://db2062spq951s.cloudfront.net"
-chs_url = "https://find-and-update.company-information.service.gov.uk"
-cic_fee = 15
-cookie_domain = "company-information.service.gov.uk"
-cookie_name = "__SID"
+use_set_environment_files = true
 desired_task_count = 2 # use multi instance in live
-feedback_link = "https://www.smartsurvey.co.uk/s/file-package-accts-fdbk/"
 log_level = "info"
 node_env = "production"
-overseas_fee = 33
 piwik_url = "https://matomo.companieshouse.gov.uk"
-submit_validation_url = "https://test-validator.companieshouse.gov.uk/xbrl_validate/submit"
 tz = "Europe/London"
```

Comment:
node_env, piwik_ur, tz and desired_task_count should also be removed and provided by the env file